blog/2022/11-18-tagging-as-a-treat/index.md

```diff
@@ -0,0 +1,143 @@
+---
+authors: [anja]
+tags: [aws, tagging, cloud]
+---
+
+# Tagging as a Treat
+
+Cloud Tagging Strategies and Policies are hailed as one of the most efficient ways to keep your cloud infrastructure controllable. But are they really?
```

Comment:
Is "Cloud Tagging Strategies and Policies" a proper noun? 🤔

Reply:
yea no